test/03-date-tests.js

```diff
@@ -87,6 +87,14 @@ describe('03-date-tasks', function() {
                 startDate: new Date(2000,1,1,10,0,0),
                 endDate:   new Date(2000,1,2,15,20,10,453),
                 expected:  '29:20:10.453'
+            },{
+                startDate: new Date(2000,1,1,10,0,0),
+                endDate:   new Date(2000,2,2,15,20,10,453),
+                expected:  '725:20:10.453'
+            },{
+                startDate: new Date(2000,1,1,10,0,0),
+                endDate:   new Date(2001,1,2,15,20,10,453),
+                expected:  '8813:20:10.453'
             }
```

Comment:
Why did you add additional test cases? is it really important?

Reply:
I just wanted to check, and forgot to delete it later.